crates/repl_ui/src/repl_state.rs

```diff
@@ -275,6 +275,9 @@ pub fn parse_src<'a>(arena: &'a Bump, line: &'a str) -> ParseOutcome<'a> {
 
             match roc_parse::expr::loc_expr(true).parse(arena, State::new(src_bytes), 0) {
                 Ok((_, loc_expr, _)) => ParseOutcome::Expr(loc_expr.value),
+                Err((roc_parse::parser::Progress::MadeProgress, EExpr::Start(_))) => {
```

Comment:
Not sure about this pattern. It fits the problem, but it can potentially match smth not undesirable

Reply:
Seems okay for now, thanks for doing this